scripts/jenkins/qa_devstack.sh

```diff
@@ -92,6 +92,27 @@ function h_setup_extra_disk {
     mount /dev/vdb /opt/stack
 }
 
+function h_install_devstack_deps {
+    # Translated package dependencies from Ubuntu Devstack
+    # plus some extra stuff the image on ci1 is missing.
+    $zypper in \
+               bridge-utils \
+               gettext-runtime \
+               iptables \
+               libffi48-devel \
+               libmysqlclient-devel \
+               libopenssl-devel \
+               libpqxx-devel \
+               libxml2-devel \
```

Comment:
usually this and the next one are compiled when python-devel is present, and IIRC this is installed via pip in devstack